.github/workflows/build_and_test.yml

```diff
@@ -0,0 +1,75 @@
+name: Build & Test
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    tags:
+      - r[0-9]+.[0-9]+
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}-${{ github.event_name == 'workflow_dispatch' }}-${{ github.event_name == 'schedule' }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: "Build XLA"
+    uses: ./.github/workflows/_build.yml
+    with:
+      ecr-docker-image-base: 308535385114.dkr.ecr.us-east-1.amazonaws.com/pytorch/xla_base
+      gcr-docker-image: gcr.io/tpu-pytorch/xla_base:latest
+    secrets:
+      gcloud-service-key: ${{ secrets.GCLOUD_SERVICE_KEY }}
+
+  test-cpu:
+    name: "CPU tests"
+    uses: ./.github/workflows/_test.yml
+    needs: build
+    with:
+      docker-image: ${{ needs.build.outputs.docker-image }}
+    secrets:
+      gcloud-service-key: ${{ secrets.GCLOUD_SERVICE_KEY }}
+
+  test-cuda:
+    name: "GPU tests"
+    uses: ./.github/workflows/_test.yml
+    needs: build
+    with:
+      docker-image: ${{ needs.build.outputs.docker-image }}
+      runner: linux.8xlarge.nvidia.gpu
+    secrets:
+      gcloud-service-key: ${{ secrets.GCLOUD_SERVICE_KEY }}
+
+  test-cpu-coverage:
+    name: "Collect CPU test coverage"
+    if: github.event_name == 'push' && github.event.ref == 'master'
+    uses: ./.github/workflows/_test.yml
+    needs: build
+    with:
+      docker-image: ${{ needs.build.outputs.docker-image }}
+      collect-coverage: true
+    secrets:
+      gcloud-service-key: ${{ secrets.GCLOUD_SERVICE_KEY }}
+
+  test-gpu-coverage:
+    name: "Collect GPU test coverage"
+    if: github.event_name == 'push' && github.event.ref == 'master'
+    uses: ./.github/workflows/_test.yml
+    needs: build
+    with:
+      docker-image: ${{ needs.build.outputs.docker-image }}
+      runner: linux.8xlarge.nvidia.gpu
+      collect-coverage: true
+    secrets:
+      gcloud-service-key: ${{ secrets.GCLOUD_SERVICE_KEY }}
+
+  push-docs:
+    name: "Build & ublish docs"
```

Comment:
```suggestion
    name: "Build & publish docs"
```

Reply:
I know, but I'm waiting for a review and clear signal before pushing the fix for the typo